src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -484,21 +484,21 @@ internal MulticlassClassificationTrainers(MulticlassClassificationCatalog catalo
         /// <param name="labelColumnName">The name of the label column in <paramref name="data"/>.</param>
         /// <param name="scoreColumnName">The name of the score column in <paramref name="data"/>.</param>
         /// <param name="predictedLabelColumnName">The name of the predicted label column in <paramref name="data"/>.</param>
-        /// <param name="topK">If given a positive value, the <see cref="MulticlassClassificationMetrics.TopKAccuracy"/> will be filled with
+        /// <param name="topPredictionCount">If given a positive value, the <see cref="MulticlassClassificationMetrics.TopKAccuracy"/> will be filled with
```

Comment:
Did you also rename `TopKAccuracy`? If we make this change, we won't have parallelism with the evaluation metrics.

Reply:
We don't need to. `TopKAccuracy` is as good as `Accuracy`. This original `topK` is a parameter when computing `TopKAccuracy`, not `TopKAccuracy` itself.

Reply:
Discussed offline. We will do `topKPredictionCount` to associate it with `TopKAccuracy`.